### PR TITLE
docs(tutorials): add TEMxServerless tutorial 

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

File: tutorials/send-emails-tem-serverless-framework/index.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        export SCW_SECRET_KEY=<Scaleway secret key>
+        export SCW_DEFAULT_ORGANIZATION_ID=<Scaleway default Organization>
+        export SCW_DEFAULT_PROJECT_ID=<Scaleway default Project>
+        export SCW_DEFAULT_REGION="fr-par"
+        export SCW_API_URL="https://api.scaleway.com"
+        ```
+
+## Add your variables
+
+1. In the `tem` repository, open the `serverless.yml` file and replace `$SCW_SECRET_KEY`, `$SCW_PROJECT_ID`, `$SCW_REGION` in the `provider` section with your own variables.
```
Comment (thomas-tacquet) [inline tutorials/send-emails-tem-serverless-framework/index.mdx:54]: For secret management this is not the recommanded way to do it, we encourage to use env var here: https://github.com/scaleway/serverless-scaleway-functions/blob/master/docs/secrets.md  and make the same change in the file under this comment 😇

File: tutorials/send-emails-tem-serverless-framework/index.mdx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+## Deploy your function
+
+1. In the same terminal as the previous steps, copy and paste the following command to trigger the deployment of your function:
+        ```
+        serverless deploy
+        ```
+2. Log in to your Scaleway account and click the Functions section in the left-side menu. Your `gomail` namespace displays.
+3. Click the `gomail` namespace, then click the `gomail` function.
+4. Click the **Endpoints** tab and copy your function's domain. Make sure you save it as you will need it in the next steps.
+5. Click the **Security** tab, then click **+ Generate token**.
```
Comment (thomas-tacquet) [inline tutorials/send-emails-tem-serverless-framework/index.mdx:99]: I think that you can link this help page: https://www.scaleway.com/en/docs/serverless/functions/how-to/secure-a-function/#restrict-access-to-your-functions

File: tutorials/send-emails-tem-serverless-framework/index.mdx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+## Install and clone the necessary tools
+
+1. Open a terminal and paste the following command to install the Serverless Framework CLI on your local machine:
+
+        ```
+        npm install serverless -g
+        ```
+2. Clone the GitHub repository containing the configuration files needed to deploy your function:
+        ```
+        git clone !!!ADD REPO HERE!!
```
Comment (thomas-tacquet) [inline tutorials/send-emails-tem-serverless-framework/index.mdx:35]: I'll send you the link

Comment (thomas-tacquet) [reply tutorials/send-emails-tem-serverless-framework/index.mdx:35]: Here is the link + command to run: 

```console
git clone git@github.com:scaleway/serverless-examples.git
cd functions/go-mail
```

**Review by bene2k1 — APPROVED** [review]

**Review by thomas-tacquet — APPROVED** [review]

**Review by SamyOubouaziz — APPROVED** [review]

**Pull request merged**